### ISSUE TITLE
Selected rotation is not saved when exiting the program

### ISSUE THREAD

**Issue by veluria** [timhul/ClassicSim#210]

**Comment by timhul** [comment]
Yep, one of those things I've known about forever but due to some.. sub-optimal design choices this is actually surprisingly difficult to save (or was the last time I tried).
